### PR TITLE
Modified script-task Docu

### PR DESCRIPTION
**Pull request by Hafflgav**
Fixed variable mapping link & added zeebe script worker community extension.

### CODE REVIEW

File: docs/components/modeler/bpmn/script-tasks/script-tasks.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ Jobs for script tasks are not processed by Zeebe itself. To process them, provid
 
 ## Defining a task
 
-A script task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition). It specifies
+A script task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition) the same way as a service task does. It specifies
 the type of job workers should subscribe to (e.g. `script`).
 
 Use [task headers](/components/modeler/bpmn/service-tasks/service-tasks.md#task-headers) to pass static parameters to the job
-worker (e.g. the script to evaluate).
+worker (e.g. the script to evaluate). The community extension [Zeebe Script Worker](https://github.com/camunda-community-hub/zeebe-script-worker) requires certain attributes to be set in the task headers.
```
Comment (korthout) [inline docs/components/modeler/bpmn/script-tasks/script-tasks.md:32]: 👍 I like this change

💭  Do we have somewhere documentation about task headers that we could link to here?

File: docs/components/modeler/bpmn/script-tasks/script-tasks.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ Jobs for script tasks are not processed by Zeebe itself. To process them, provid
...
 
-Define [variable mappings](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings) to transform the
+Define [variable mappings](/components/concepts/variables.md#inputoutput-variable-mappings) to transform the
```
Comment (korthout) [inline docs/components/modeler/bpmn/script-tasks/script-tasks.md:34]: ❌  I have the same concern here as I mentioned [here](https://github.com/camunda-cloud/camunda-cloud-documentation/pull/602#discussion_r806036713). Please reconsider.

**Review by korthout — CHANGES_REQUESTED** [review]
Generally looks good, but there's a concern that should be addressed before merging.

File: docs/components/modeler/bpmn/script-tasks/script-tasks.md (hunk 2 above)
Comment (Hafflgav) [reply docs/components/modeler/bpmn/script-tasks/script-tasks.md]: I reconsidered

File: docs/components/modeler/bpmn/script-tasks/script-tasks.md (hunk 1 above)
Comment (Hafflgav) [reply docs/components/modeler/bpmn/script-tasks/script-tasks.md:32]: Isn't documentation about task headers linked in line 31? Or are you thinking about something different here?

Comment (korthout) [reply docs/components/modeler/bpmn/script-tasks/script-tasks.md:32]: Haha, yes you're absolutely right! I missed that one 😅

Comment (Hafflgav) [reply docs/components/modeler/bpmn/script-tasks/script-tasks.md:32]: Okay - anything else missing? 
(I am still in need for an approval here)

**Pull request merged**